README.md

```diff
@@ -153,6 +153,8 @@ Some [Content Management System](#content-management-systems-cms) solutions also
 - [Huginn](https://github.com/cantino/huginn) - Allows you to build agents that monitor and act on your behalf. `MIT` `Ruby`
 - [Http2pic](https://http2pic.haschek.at/) - Website screenshots/renderer. It uses the wkhtmltox to render websites with various options. ([Source Code](https://github.com/chrisiaut/http2pic)) `Apache 2.0` `PHP/Javascript`
 - [Kibitzr](https://kibitzr.github.io) - Lightweight personal web assistant with powerful integrations. ([Source Code](https://github.com/kibitzr/kibitzr/)) `MIT` `Python`
+- [LazyLibrarian](https://github.com/DobyTang/LazyLibrarian) - LazyLibrarian is a program to follow authors and grab metadata for all your digital reading needs. It uses a combination of Goodreads Librarything and optionally GoogleBooks as sources for author info and book info ([Source Code](https://github.com/DobyTang/LazyLibrarian)) `GPL-3.0` `Python`
```

Comment:
Missing full-stop at the end of the descritption.
Add ⚠ as it appears to require goodreads and librarything

Reply:
added period. 

Those are API's it uses to scrape data, not dependencies.

Reply:
Does Lazy Librarian function with no access to Goodreads, Librarything, or Google Books? If so is it a crippling user experience, so bad that the devs consider it effectively mandatory?